### ISSUE TITLE
Steam deck controls don't 

### ISSUE THREAD

**Issue by yunnieXx** [4ian/GDevelop#7250]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/4ian/GDevelop/issues)

### Describe the bug

When attempting to use a Steam Deck to control a game in preview mode, the controller inputs are not recognized. This issue prevents proper testing of games on the Steam Deck in preview mode.

https://github.com/user-attachments/assets/bd5a1712-66b0-42d2-99b5-76cf969c6376

__

### Steps to reproduce

1. Create a new project in GDevelop.
2. Add events and actions to respond to controller inputs.
3. Switch to Preview Mode.
4. Attempt to control the game using the Steam Deck's controls.

### GDevelop platform

Desktop

### GDevelop version

5.4.219

### Platform info

<details>

*OS (e.g. Windows, Linux, macOS, Android, iOS)*
>Linux

*OS Version (e.g. Windows 10, macOS 10.15)*
>SteamOS 3.6.20 (Based on Arch Linux)

*Browser(For Web) (e.g. Chrome, Firefox, Safari)*
>

*Device(For Mobile) (e.g. iPhone 12, Samsung Galaxy S21)*
> Steam Deck

</details>

**Comment by 4ian** [comment]
Can you try opening this page and see if controls are recognized? https://luser.github.io/gamepadtest/

**Comment by yunnieXx** [comment]
> Can you try opening this page and see if controls are recognized? https://luser.github.io/gamepadtest/

Yes, my steam deck gamepad is being correctly recognized as a Xbox 360 controller.

https://github.com/user-attachments/assets/89aa5d46-6ef0-4c34-8e13-79e62e1ce84f

**Comment by Adam-Morton** [comment]
Hey @AyItzMatt and @4ian, not sure if this is still being looked into but I found a workaround to make the Steam Deck controls work in Preview. 

Download Gdevelop from the Discover app, then download the AppImage file from github for the latest version, and launch the AppImage file.

THAT method of launching Gdevelop allows the Steam Deck controls to work properly, but launching the app that was installed via Discover won't have working Steam Deck controls.

**Comment by 4ian** [comment]
Interesting, thanks for letting us know! Do you have a screenshot of "Discover" you could share to see exactly what this is on the Steam Deck?

**Comment by Adam-Morton** [comment]
For sure - it's like the App Store for Linux programs, so I initially found GDevelop in here, but have since been downloading the App Image directly and that's been making the Steam Controls work in preview.

Here's the Discover app and the GDevelop page for it:

![Image](https://github.com/user-attachments/assets/8d11eced-a799-4b97-8ec1-ec7b005c0081)

**Comment by 4ian** [comment]
I see! This is probably the same as the Flathub package: https://flathub.org/apps/io.gdevelop.ide

It's based on the .deb package as it can be seen on the repository that automatically build the Flatpak: https://github.com/flathub/io.gdevelop.ide/pull/79/files

So I guess there is some difference - maybe in the way permissions are handled? - between AppImages and the .deb package.

Thanks for sharing in any case!

**Comment by Adam-Morton** [comment]
No problem! Glad to help with some info, and hopefully if anyone else finds this issue they see the workaround :)
